docker-compose.yaml

```
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  docker:
+    image: docker:dind
+    privileged: true
+    environment:
+      DOCKER_TLS_CERTDIR: /certs
+    networks:
+      - docker
+    volumes:
+      - docker-certs-ca:/certs
+      - docker-certs-client:/certs/client
+  grype:
+    build:
+      context: .
+      dockerfile: local_dev.Dockerfile
```

nit: could we rename this?
```suggestion
      dockerfile: dev.Dockerfile
```